### ISSUE TITLE
Hide Admin / Mod tags unless using "speak as moderator" (distinguishing post)

### ISSUE THREAD

**Issue by Crashdoom** [LemmyNet/lemmy-ui#1828]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

Currently, Lemmy always shows the "M" (Mod) and "A" (Admin) tags on posts and comments made by moderators and instance admins respectively which can be off-putting to some users and also discourages mods and admins from taking part in discussions for fear of their thoughts and opinions being construed as an official statement.

I'd like to see those tags hidden by default, unless the post or comment is distinguished using the existing "speak as moderator" button. Similar to how moderators and admins on Reddit can take part in individual discussions without their comments constantly being marked as "speaking officially."

I don't believe this is a core Lemmy issue as the flag to "distinguish" a post is available via the API request made by lemmy-ui, but that this change would make https://github.com/LemmyNet/lemmy/issues/3066 more relevant to help identify when a post is being made "officially" by a mod / admin.

**Comment by vinoff** [comment]
I am a mod in a sub and I find the ability to toggle the Mod tag a must have! I am considering creating a secondary account just so I can post in the sub I mod without it seeing like I am spamming too much. 

I find this specially important in the current stage of Lemmy, in which it is extremely important to create engaging content. Mods are usually those who post more at the initial phase of a community. Currently, I am a bit scared of doing that with all the "Mod" tags attached.

**Comment by dessalines** [comment]
> I am a mod in a sub and I find the ability to toggle the Mod tag a must have! I am considering creating a secondary account just so I can post in the sub I mod without it seeming like I am spamming too much

To me it seems not-transparent to hide the fact that you're actually a mod / admin, to others, and have extra abilities that they don't. Showing the mod / admin badge is a deliberate choice to be transparent to everyone who engages with you.

**Comment by Crashdoom** [comment]
> > I am a mod in a sub and I find the ability to toggle the Mod tag a must have! I am considering creating a secondary account just so I can post in the sub I mod without it seeming like I am spamming too much
> 
> To me it seems not-transparent to hide the fact that you're actually a mod / admin, to others, and have extra abilities that they don't. Showing the mod / admin badge is a deliberate choice to be transparent to everyone who engages with you.

The transparency issue seems like a bit of a moot point since the mods are listed on the sidebar, same with admins of an instance. A lot of mods and admins don't like the tag following them around constantly as it discourages users from treating them like users and discourages discussion for fear of retribution; If something rises to the need for mod / admin intervention, the distinguish flag is there for a reason.

Allowing mods / admins to hide the tag when not doing "official" business allows them to be "off-duty" and have opinions of their own, not having to worry about if that makes their opinion official policy of the instance / community.

Otherwise, the distinguish feature is entirely pointless as all posts are already "distinguished" with the bright red admin and green moderator tags, and those posts will be assumed to be made in an official capacity.

**Comment by dessalines** [comment]
> The transparency issue seems like a bit of a moot point since the mods are listed on the sidebar, same with admins of an instance.

That is true, just like a bio or other extra info, this could be behind a single layer.

[Still tho, see this comment](https://github.com/LemmyNet/lemmy-ui/pull/2069#issuecomment-1699688003). Its a matter of perspective: do we care more about the user who's engaging with someone who has real power over them? Or do we care more about a moderator's preference to hide those abilities?
